test/lib/usd/translators/testUsdExportImportRoundtripPreviewSurface.py

```diff
@@ -68,6 +68,7 @@ def testUsdPreviewSurfaceRoundtrip(self):
         cmds.setAttr(material_node + ".specularColor", 0.125, 0.25, 0.75,
                      type="double3")
         cmds.setAttr(material_node + ".useSpecularWorkflow", True)
+        cmds.setAttr("{}.opacityThreshold".format(material_node), 0.5)
```

Comment:
Sorry, this is a really obnoxious nit-pick, but I'd stick with the convention already in use here and use `+` rather than `format()`. I actually prefer what you used personally, but just so someone looking at this code later doesn't wonder whether there's a reason it's treated differently from the other attributes:
```Python
cmds.setAttr(materialNode + ".opacityThreshold", 0.5)
```